services/web-app/src/lib/types.ts

```diff
@@ -1,8 +1,8 @@
 export interface User {
-    email: "" ,
-    userId: "",
-    apiKey: "",
-    name: "",
-    pictureUrl?: "",
+    email: string ,
+    userId: string,
+    apiKey: string,
+    name: string,
+    pictureUrl?: string,
```

Comment:
[Note] This is a bug- by setting "" values here you actually setting constants rather than default values- so if you tried to do 
```
const user: User;
user.name = 'Adam' 
```
You would see an error as user.name can only be set to "".